frontend/rpc.bal

```diff
@@ -18,56 +18,58 @@ import ballerina/grpc;
 import ballerina/log;
 import ballerina/random;
 
-configurable string currencyHost = "localhost";
+const string LOCALHOST = "localhost";
+
+configurable string currencyHost = LOCALHOST;
 @display {
-    label: "",
+    label: "Currency",
     id: "currency"
 }
 final CurrencyServiceClient currencyClient = check new ("http://" + currencyHost + ":9093");
 
-configurable string catalogHost = "localhost";
+configurable string catalogHost = LOCALHOST;
 @display {
-    label: "",
+    label: "Catalog",
     id: "catalog"
 }
 final ProductCatalogServiceClient catalogClient = check new ("http://" + catalogHost + ":9091");
 
-configurable string cartHost = "localhost";
+configurable string cartHost = LOCALHOST;
 @display {
-    label: "",
+    label: "Cart",
     id: "cart"
 }
 final CartServiceClient cartClient = check new ("http://" + cartHost + ":9092");
 
-configurable string shippingHost = "localhost";
+configurable string shippingHost = LOCALHOST;
 @display {
-    label: "",
+    label: "Shipping",
     id: "shipping"
 }
 final ShippingServiceClient shippingClient = check new ("http://" + shippingHost + ":9095");
 
-configurable string recommandHost = "localhost";
+configurable string recommendHost = LOCALHOST;
 @display {
-    label: "",
+    label: "Recommendation",
     id: "recommendation"
 }
-final RecommendationServiceClient recommandClient = check new ("http://" + recommandHost + ":9090");
+final RecommendationServiceClient recommendClient = check new ("http://" + recommendHost + ":9090");
```

Comment:
Shall we convert these to string template

Reply:
Updated in https://github.com/ballerina-guides/gcp-microservices-demo/pull/30/commits/230872e6476170a9393878d3ead7d3d4619502a8